Core/Lib/ExtendedController/WidgetButton.php

```diff
@@ -245,6 +245,24 @@ private function getModalHTML($label, $class = 'col-sm-auto')
         return $html;
     }
 
+    /**
+     * Return the HTML code to display a button with javascript Action
+     * 
+     * @param string $label
+     * @param string $hint
```

Comment:
$hint not used, please remove.

Reply:
Maybe it would be interesting if you could indicate the hint and apply the popup that is being used in other buttons